src/main/java/com/autotune/analyzer/deployment/AutotuneDeploymentInfo.java

```diff
@@ -150,8 +150,7 @@ public static void setMonitoringAgent(String monitoringAgent) throws MonitoringA
         if (AutotuneSupportedTypes.MONITORING_AGENTS_SUPPORTED.contains(monitoringAgent)) {
             AutotuneDeploymentInfo.monitoringAgent = monitoringAgent;
         } else {
-            LOGGER.error("Monitoring agent {}  is not supported", monitoringAgent);
-            throw new MonitoringAgentNotSupportedException();
+            LOGGER.warn("Monitoring agent is mandatory for Local Monitoring/Experiment mode.");
```

Comment:
We still need to know what was the `monitoringAgent` specified. In case a monitoring agent that is not supported, we need to mention that